packages/tests/server/react/polymorphism.factory.tsx

```diff
@@ -33,10 +29,6 @@ export type FileExportStatusType = z.infer<typeof FileExportStatus>;
 // Dependencies
 //
 
-type BaseProcedure<TRoot extends AnyRootTypes> = ReturnType<
```

Comment:
Interestingly what was here before actually does produce a type with typeof unsetMarker there, so inter-compat with .input merging wouldn't work in production today. Maybe we can find a compromise then

Reply:
I tried export this from @trpc/server basically and found the same problem popped up with procedure.input() being passed through
```
// type $RootTypes = ReturnType<typeof initTRPC.create>['_config']['$types'];
// export type AnyTRPCProcedureBuilder = ReturnType<
//   typeof createBuilder<$RootTypes['ctx'], $RootTypes['meta']>
// >;
```